src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -508,42 +515,42 @@ public sealed class BinaryClassificationMetricsStatistics : MetricsStatisticsBas
         /// <summary>
         /// Summary statistics for AUPRC.
```

Comment:
Area under the Precision-Recall Curve?

Reply:
Yes, will do `<see` against the relevant original metrics classes so that the relationship is a bit more explicit.